src/algorithms/reco/InclusiveKinematicsSigma.cc

```diff
@@ -88,6 +82,7 @@ namespace eicrecon {
     auto sigma_tot = sigma_e + sigma_h;
 
     // Calculate kinematic variables
+    const auto m_proton = m_particleSvc.particle(2212).mass;
```

Comment:
```suggestion
    static const auto m_proton = m_particleSvc.particle(2212).mass;
```

Reply:
This is arguably too narrow, since it assumes scattering off the proton.